roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -9,14 +9,34 @@
 # - openshift_release
 
 # tasks file for openshift_node_upgrade
+
+- name: Stop rpm based services
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+  - "{{ openshift.common.service_type }}-node"
+  - openvswitch
+  failed_when: false
+  when: not openshift.common.is_containerized | bool
+
+- name: Stop additional containerized services
+  service: name={{ item }} state=stopped
+  with_items:
+    - "{{ openshift.common.service_type }}-master"
+    - "{{ openshift.common.service_type }}-master-api"
+    - "{{ openshift.common.service_type }}-master-controllers"
+    - etcd_container
+  failed_when: false
+  when: openshift.common.is_containerized | bool
```

Comment:
updated to remove the conditional so it now stops node and openvswitch no matter what, then conditionally stops additional services for containerized